sonar-application/src/main/assembly/conf/sonar.properties

```diff
@@ -121,6 +121,12 @@
 # The default value is 25.
 #sonar.web.http.acceptCount=25
 
+# The number of milliseconds this Connector will wait for another HTTP request before closing the 
+# connection. The default value is to use the value that has been set for the connectionTimeout 
+# attribute. Use a value of -1 to indicate no (i.e. infinite) timeout.
+# The default value is 25.
```

Comment:
Hi @jasondamour,
Thank you for your investigation and contribution.
We are debating and testing your solution. In the meantime could you fix the comment here? The default value is not 25.